pkg/kopia/command/const.go

```diff
@@ -75,4 +75,16 @@ const (
 
 	// Repository specific
 	repositorySubCommand = "repository"
+
+	// DefaultCacheDirectory is the directory where kopia content cache is created
+	DefaultCacheDirectory = "/tmp/kopia-cache"
+
+	// DefaultConfigFilePath is the file which contains kopia repo config
+	DefaultConfigFilePath = "/tmp/kopia-repository.config"
+
+	// DefaultConfigDirectory is the directory which contains custom kopia repo config
+	DefaultConfigDirectory = "/tmp/kopia-repository"
+
+	// DefaultLogDirectory is the directory where kopia log file is created
+	DefaultLogDirectory = "/tmp/kopia-log"
```

Comment:
Add these constants as these are being heavily used like kopia.DefaultConfigFilePath in implementation of wrappers